pkg/k8sutils/k8sutils.go

```diff
@@ -29,7 +29,7 @@ const (
 	// DefaultAdminNamespace - default admin namespace, where stork will be installed
 	DefaultAdminNamespace = "kube-system"
 	// StorkConfigMapName holds any generic config specific to stork module
-	StorkConfigMapName = "stork-objLock-config"
+	StorkConfigMapName = "stork-objlock-config"
```

Comment:
can we use one config map for all stork related configuration like kdmp,version and objectlog. maintaining multiple configmap will be troublesome. Lets setup call and sync up online since kdmp,version being used by px-backup already.

For now lets this get merged in

Reply:
Agreed, In fact this object lock config option is only for testing purpose. Customer is not going to use it. That we can add it in kdmp_config configmap itself and ask QA to use it.

Reply:
My initial thought was same, but then I didn't have much context of other config and we needed this to be in little early. Hence went that path. Particularly in "stork-config"  config map there is something related to stork-scheduler exist which need to be discussed. But I totally agree to merge them all.